bouncer/src/repo/core/model/bson/repo_bson_factory.h

```diff
@@ -322,13 +343,28 @@ namespace repo {
 					const std::vector<repo::lib::RepoVector3D>& normals,
 					const std::vector<std::vector<float>>& boundingBox,
 					const std::vector<std::vector<repo::lib::RepoVector2D>>& uvChannels,
-					const std::vector<repo_color4d_t>& colors,
 					const std::vector<float> mappingIds,
 					const std::vector<repo_mesh_mapping_t>& mappings,
 					const repo::lib::RepoUUID& id,
 					const repo::lib::RepoUUID& sharedId
 				);
 
+				static SupermeshNode makeSupermeshNode(
```

Comment:
if we rejig the param orders then we can make mappingId optional and have one function declaration?

Reply:
Done!